### PR TITLE
Progress Report 2019 Q1: Various wording improvements

### PR DESCRIPTION
**Pull request by RoadrunnerWMC**

### CODE REVIEW

File: site/content/entry/citra-progress-report-2019-q1.md [hunk 1]
```diff
@@ -165,149 +165,153 @@ Pinged users will get a notification on their desktop and the message will be hi
...
+While the current capabilities of scripting are limited to reading and writing memory at this point in time, we plan to expand this in the near future, so stay tuned!
 
 ### Status of Official Android App ([#4324](https://github.com/citra-emu/citra/pull/4324), [#4450](https://github.com/citra-emu/citra/pull/4450), [#4575](https://github.com/citra-emu/citra/pull/4575))
 
-When developer SachinVin unofficially ported Citra to Android, our own team members [BreadFish64](https://github.com/BreadFish64) and [jroweboy](https://github.com/jroweboy) were secretly working on an official port.
-Since the app was based on a few months old Citra code, it didn't contain many features, fixes, and performance improvements that were present in latest Citra code.
-This unoffical port, also heavily borrowed UI from Dolphin's android app, which didn't fully meet the requirements of Citra.
+When developer [SachinVin](https://github.com/SachinVin) unofficially ported Citra to Android, our own team members [BreadFish64](https://github.com/BreadFish64) and [jroweboy](https://github.com/jroweboy) were secretly working on an official port.
```
Comment (CaptV0rt3x) [inline site/content/entry/citra-progress-report-2019-q1.md]: Not linking to the unofficial dev's GitHub was an intentional change, as we don't want to redirect users towards the unofficial app.

Comment (RoadrunnerWMC) [reply site/content/entry/citra-progress-report-2019-q1.md]: Ah, I did wonder about that. I'll change it back, then.

Comment (jroweboy) [reply site/content/entry/citra-progress-report-2019-q1.md]: oh geez, i didn't catch this the first time. i wasn't working on android myself. i just did a GLES backend for the dank webgl and citra emscripten memes.

File: site/content/entry/citra-progress-report-2019-q1.md [hunk 2]
```diff
@@ -7,156 +7,156 @@ coauthor = "jroweboy"
...
-Hey there Citra fans! These past 6 months have been crazily exciting, and simply wonderful for Citra overall.
+Hey there, Citra fans! These past 6 months have been crazily exciting, and simply wonderful for Citra overall.
 We've made major breakthroughs which were long overdue.
 New features, bug fixes, performance improvements, improved game compatibility, and much more await you.
-So, buckle up and enjoy the ride!
+So buckle up and enjoy the ride!
 <!--more-->
 ****
-To start off - Back in August, we asked our patrons to vote on what they'd like to see us work on the most and now it's time to review how much progress we've made on everything since then.
+To start off: back in August, we asked our patrons to vote on what they'd like to see us work on the most, and now it's time to review how much progress we've made on everything since then.
```
Comment (jroweboy) [inline site/content/entry/citra-progress-report-2019-q1.md]: change the `:` to a `,` https://www.grammarbook.com/punctuation/colons.asp

Comment (RoadrunnerWMC) [reply site/content/entry/citra-progress-report-2019-q1.md]: All right.

File: site/content/entry/citra-progress-report-2019-q1.md [hunk 3]
```diff
@@ -165,149 +165,153 @@ Pinged users will get a notification on their desktop and the message will be hi
...
-Since the app was based on a few months old Citra code, it didn't contain many features, fixes, and performance improvements that were present in latest Citra code.
-This unoffical port, also heavily borrowed UI from Dolphin's android app, which didn't fully meet the requirements of Citra.
+Since the app was based on Citra code a few months old, it was missing many features, fixes, and performance improvements that were present in the latest Citra code.
+This unoffical port also heavily borrowed UI from Dolphin's Android app, which didn't fully meet the requirements of Citra.
 
-However, since then, [BreadFish64](https://github.com/BreadFish64) has been working along with [jroweboy](https://github.com/jroweboy) and [liushuyu](https://github.com/liushuyu), tirelessly to officially port Citra for Android.
+However, since then, [BreadFish64](https://github.com/BreadFish64) has been working along with [jroweboy](https://github.com/jroweboy) and [liushuyu](https://github.com/liushuyu) tirelessly to officially port Citra to Android.
```
Comment (CaptV0rt3x) [inline site/content/entry/citra-progress-report-2019-q1.md]: Could you emphasize the fact that we aren't using Dolphin's UI and are designing our app UI from scratch, by adding one more sentence ?

A user told me, that it wasn't clear.

Comment (RoadrunnerWMC) [reply site/content/entry/citra-progress-report-2019-q1.md]: Sure.

**Pull request merged**